src/Pyz/Zed/DataImport/Business/CombinedProduct/ProductPrice/Writer/CombinedProductPricePropelDataSetWriter.php

```diff
@@ -16,12 +16,19 @@
 class CombinedProductPricePropelDataSetWriter extends ProductPricePropelDataSetWriter
 {
     protected const COLUMN_ABSTRACT_SKU = CombinedProductPriceHydratorStep::COLUMN_ABSTRACT_SKU;
```

Comment:
var string

Reply:
I don't think we will fix such cases
if sniffer is okay with that, then we also good
we have potentially hundreds of such cases and time expenses are too big

Reply:
discussed with @DmitryLymarenko